app/js/app.js

```diff
@@ -1,4 +1,4 @@
-import MainController from 'js/controllers/main';
+import MainController from './controllers/main';
```

Comment:
Two reasons:
* We use relative paths in the rest of the code - so for the sake of consistency;
* Rollup considers non-relative paths as external paths and doesn't include them into bundle (sounds reasonable)